drift_dev/pubspec.yaml

```diff
@@ -1,6 +1,6 @@
 name: drift_dev
 description: Dev-dependency for users of drift. Contains a the generator and development tools.
-version: 2.4.0-dev
+version: 2.5.0-dev
```

Comment:
I'm not to sure about this one, what do you want the version number to be?

Reply:
Let's leave it as `2.4.0-dev` - I just use the `-dev` suffix as a reminder that this state hasn't been released yet. In the end, all new features will be released as part of `2.4.0`.